components/scream/src/physics/shoc/tests/shoc_pblintd_height_tests.cpp

```diff
@@ -102,7 +102,7 @@ struct UnitWrap::UnitTest<D>::TestPblintdHeight {
 
     // Save result for checking with next test
     Real pblh_save[shcol];
-    for(Int s = 0; s < shcol-1; ++s) {
+    for(Int s = 0; s < shcol; ++s) {
```

Comment:
@bogensch , I think this was a simple copy/paste error and you did not actually intend to cut this loop off 1 iteration early.

Reply:
Good catch Jim, yes this was a copy/paste error.